tests/models/test_torchscript.py

```diff
@@ -31,25 +31,43 @@ def test_torchscript_input_output(modelclass):
     model = modelclass()
     script = model.to_torchscript()
     assert isinstance(script, torch.jit.ScriptModule)
+
     model.eval()
-    model_output = model(model.example_input_array)
-    script_output = script(model.example_input_array)
+    with torch.no_grad():
+        model_output = model(model.example_input_array)
+        script_output = script(model.example_input_array)
```

Comment:
```suggestion
        model_output = model(model.example_input_array)
    script_output = script(model.example_input_array)
```
this should be sufficient

Reply:
yeah.